packages/resolve-command/src/index.js

```diff
@@ -1,72 +1,83 @@
+import 'regenerator-runtime/runtime';
+
 function verifyCommand(command) {
```

Comment:
Unfortunately underlying Promise uses direct resolve/reject method, which are used on callbacks of Stream API functions, so it can't be refactored to async/await :(